client/pyproject.toml

```diff
@@ -0,0 +1,114 @@
+[project]
+# https://packaging.python.org/en/latest/specifications/declaring-project-metadata/
+name = "rubin-nublado-client"
+description = "Client for JupyterHub/JupyterLab in the Rubin Science Platform context"
+license = { file = "LICENSE" }
+readme = "README.md"
+keywords = ["rubin", "lsst"]
+# https://pypi.org/classifiers/
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.12",
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "Operating System :: POSIX",
+    "Typing :: Typed",
+]
+requires-python = ">=3.12"
+# Use requirements/main.in for runtime dependencies instead.
+dependencies = []
```

Comment:
Ah, this doesn't work for Python packages, only for our Docker-based apps. You'll need to put dependencies into the `pyproject.toml` file.

Reply:
Theoretically dev.in/.txt are fine with Python packages, but there's no point having a main.in/.txt file.

Reply:
Done.